### PR TITLE
DRIVERS-2855: Convert default write concern tests to unified format

### PR DESCRIPTION
**Pull request by jmikola**
https://jira.mongodb.org/browse/DRIVERS-2855

Please complete the following before merging:

- [ ] Update changelog.
- [x] Make sure there are generated JSON files from the YAML test files.
- [x] Test changes in at least one language driver.
- [x] Test these changes against all server versions and topologies (including standalone, replica set, sharded clusters, and serverless).

POC: https://github.com/mongodb/mongo-php-library/pull/1252

### CODE REVIEW

File: source/read-write-concern/tests/operation/default-write-concern-2.6.json [hunk 1]
```diff
@@ -1,19 +1,55 @@
 {
```
Comment (alcaeus) [inline source/read-write-concern/tests/operation/default-write-concern-2.6.json:1]: Nothing major, just noted that for other tests you created a `unified` directory under tests, whereas this PR keeps original files.

**Review by alcaeus — APPROVED** [review]
LGTM with a minor note on test location.

File: source/read-write-concern/tests/operation/default-write-concern-2.6.json (hunk 1 above)
Comment (jmikola) [reply source/read-write-concern/tests/operation/default-write-concern-2.6.json:1]: This was intentional, as the tests were already under an `operation/` directory. In other specs, legacy tests were either in `tests/` or, in the case of CRUD, a specific directory.

These tests are old and few enough that I'm not worried at all about people mixing up legacy and unified test files.

**Comment by jmikola** [comment]
Note: link check will be fixed by https://github.com/mongodb/specifications/pull/1549

**Comment by jmikola** [comment]
@jyemin: Do you want to POC this in Java before I merge?

**Comment by jyemin** [comment]
Risk seems low on these so go ahead without me.

**Pull request merged**